data-prepper-plugins/aws-lambda/README.md

```diff
@@ -1,4 +1,55 @@
 
+# Lambda Processor
+
+This plugin enables you to send data from your Data Prepper pipeline directly to AWS Lambda functions for further processing.
+
+## Usage
+```aidl
+lambda-pipeline:
+...
+  processor:
+    - aws_lambda:
+        aws:
+            region: "us-east-1"
+            sts_role_arn: "<arn>"
+        function_name: "uploadToS3Lambda"
+        max_retries: 3
+        invocation_type: "RequestResponse"
+        payload_model: "batch_event"
+        batch:
+            key_name: "osi_key"
+            threshold:
+                event_count: 10
+                event_collect_timeout: 15s
+                maximum_size: 3mb
+```
+
+`invocation_type` as RequestResponse will be used when the response from aws lambda comes back to dataprepper.
+`invocation_type` as Event is used when the response from aws lambda goes to an s3 bucket.
+
+In batch options, an implicit batch threshold option is that if events size is 3mb, we flush it.
+`payload_model` this is used to define how the payload should be constructed from a dataprepper event.
+`payload_model` as batch_event is used when the output needs to be formed as a batch of multiple events,
```

Comment:
are there other values for `paylod_model` ?

Reply:
fixed